sample/build.gradle

```diff
@@ -1,35 +1,33 @@
-apply from: '../dependencies.gradle'
-
-apply plugin: 'com.android.application'
-apply plugin: 'kotlin-android'
-apply plugin: 'kotlin-android-extensions'
+plugins {
+    id 'com.android.application'
+    id 'kotlin-android'
+    id 'kotlin-android-extensions'
+    id 'org.jetbrains.kotlin.android'
+}
 
+apply from: '../dependencies.gradle'
 
 android {
-  compileSdkVersion versions.compileSdk
-  buildToolsVersion versions.buildTools
+    compileSdk versions.compileSdk
+    namespace 'com.cottacush.android'
```

Comment:
We can put the package name directly in the Manifest file to avoid setting it here, since we do not manipulate package name in any fancy way.

Reply:
Actually in Gradle 7.*/8.0 above, that's now allowed anymore. It has to be defined in the gradle file.